doc/cephadm/services/monitoring.rst

```diff
@@ -328,6 +328,32 @@ Due to performance reasons, monitoring of RBD images is disabled by default. For
 :ref:`prometheus-rbd-io-statistics`. If disabled, the overview and details dashboards will stay empty in Grafana
 and the metrics will not be visible in Prometheus.
 
+Setting up Prometheus
+-----------------------
+
+Setting Prometheus Retention Time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Cephadm provides the option to set the prometheus tdsb retention time using
+a ``retention_time`` field in the prometheus service spec. The value defaults
+to 15 days (15d). If you would like a different value, such as 1 year (1y) you
+can apply a service spec similar to:
+
+.. code-block:: yaml
+
+    service_type: prometheus
+    placement:
+      count: 1
+    spec:
```

Comment:
Please, just ignore if it doesn't apply but I remember the configuration section was called something like _config_

Reply:
config is for telling cephadm to set config options, spec is for specifying fields that aren't part of the generic service spec but are included in a specific service's spec. Eg. `rgw_frontend_port` for rgw would go in the spec section.